extensions/test-node/README.md

```
@@ -17,119 +17,252 @@ pnpm add -D @tevm/test-node vitest
+await client.getBlock({ blockNumber: 1n })
+await client.server.stop() // Does not save
+
+// Manually trigger save
+await client.saveSnapshots() // Now saved
+```
+
+## Snapshot Location
+
+Snapshots are automatically placed in a `__rpc_snapshots__` subdirectory next to your test file:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)